ChangeLog

```diff
@@ -41,6 +41,7 @@
  * #1403 (Python import otagrum throws an error)
  * #1405 (Most of the simulation algorithms for rare event fail on a coronavirus example)
  * #1416 (MethodOfMomentsFactory has no setOptimizationBounds method)
+ * #1419 (BoundingVolumeHierarchy segaults/hangs)
```

Comment:
could you also take some of my changes regarding this issue ? (mainly instanciation of BVH class, see e272f93e18aa292e45c5f3998b3e0eb5027df0e6)

Reply:
done